python/TestHarness/TestHarness.py

```diff
@@ -637,17 +643,20 @@ def handleTestResult(self, tester, output, result, start=0, end=0, add_to_table=
             timing = self.getSolveTime(output)
 
         # format the SKIP messages received, otherwise leave 'result' unmolested
-        if status == 'SKIP':
+        if status == tester.bucket_skip:
             # Include caveats in skipped messages? Usefull to know when a scaled long "RUNNING..." test completes
             # but Exodiff is instructed to 'SKIP' on scaled tests.
             if len(caveats):
                 result = '[' + ', '.join(caveats).upper() + '] skipped (' + tester.getStatusMessage() + ')'
             else:
                 result = 'skipped (' + tester.getStatusMessage() + ')'
 
+        elif status == tester.bucket_deleted:
+            result = tester.getStatusMessage()
+
```

Comment:
This allows deleted tests to be printed when the extra info ( -e ) argument is supplied.